tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/helpers/serviceinstance.go

```diff
@@ -0,0 +1,28 @@
+package helpers
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	servicecatalog "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+)
+
+func CreateServiceInstance(serviceCatalogInterface servicecatalog.Interface, serviceId, namespace string) error {
+	serviceInstance := &v1beta1.ServiceInstance{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
Why do you define a TypeMeta in a Go object? TypeMeta is only necessary when serialization to/from JSON is involved.

Reply:
done